srcpkgs/portaudio/template

```diff
@@ -11,8 +11,8 @@ short_desc="Portable cross-platform audio I/O library"
 maintainer="Érico Nogueira <ericonr@disroot.org>"
 license="MIT"
 homepage="http://www.portaudio.com"
-distfiles="http://www.${pkgname}.com/archives/pa_stable_v${version%.*}_${version#*.}.tgz"
-checksum=f5a21d7dcd6ee84397446fa1fa1a0675bb2e8a4a6dceb4305a8404698d8d1513
+distfiles="http://files.portaudio.com/archives/pa_stable_v190700_20210406.tgz"
+checksum=47efbf42c77c19a05d22e627d42873e991ec0c1357219c0d74ce6a2948cb2def
```

Comment:
Version scheme change is problematic. Now one has to update version *and* distfiles. 19.7.0 to 190600.20161030 is also seen downgrade by xbps, so `reverts` with old versions would need to be added, and xbps-still would list it as downgrade.

Reply:
I changed the source location to github and added a `reverts`.